.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "server/scripts/shared"]
 	path = server/scripts/shared
 	url = git@github.com:defold/aws-build-tools.git
+
+[submodule "server/scripts/task-definitions"]
```

Comment:
The AWS task definitions are here. These now contain application.yml overrides with paths to resources we do not wish to share openly.